apps/studio/src/features/sign-in/components/SingpassLogin/SingpassLoginButton.tsx

```diff
@@ -23,24 +27,27 @@ export const SingpassLoginButton = (): JSX.Element | null => {
   }
 
   return (
-    <Stack gap="0.75rem">
-      <Button
-        colorScheme="neutral"
-        height="2.75rem"
-        size="xs"
-        variant="outline"
-        isLoading={singpassLoginMutation.isLoading}
-        onClick={handleSingpassLogin}
-        aria-label="Authenticate with Singpass"
-      >
-        <Flex align="center" flexDirection="row" aria-hidden>
-          <Text>Authenticate with </Text>
-          {/* Negative margin so the svg sits on the same line as the text */}
-          <Box mb="-3px">
-            <SingpassFullLogo height="1rem" />
-          </Box>
-        </Flex>
-      </Button>
-    </Stack>
+    <Button
+      height="2.75rem"
+      width="full"
+      size="sm"
+      variant="outline"
+      colorScheme="clear"
+      bgColor="#F4333D"
+      _disabled={{ bgColor: "#F4333D" }}
+      _hover={{ bgColor: "#B0262D" }}
+      textColor="white"
+      isLoading={singpassLoginMutation.isLoading}
+      onClick={handleSingpassLogin}
+      aria-label="Authenticate with Singpass"
+    >
+      <Flex align="center" flexDirection="row" aria-hidden>
+        <Text fontWeight="700">Authenticate with </Text>
+        {/* Negative margin so the svg sits on the same line as the text */}
+        <Box mb="-3px">
+          <SingpassFullLogo height="1rem" />
+        </Box>
+      </Flex>
```

Comment:
i'm abit confused - why not just set the mb on the `SingpassFullLogo` and why doesn't `align="center"` work? (can use `textAlign` too)

Reply:
Eh haha this is taken directly from SGID, but from what I understand it is because the logo has a left-right margin, so this is to bring it closer to make it read more naturally. The alignment is on the flex alr yeah.